inference-engine/src/low_precision_transformations/include/low_precision/reduce_base_transformation.hpp

```diff
@@ -0,0 +1,30 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <memory>
+#include <ngraph/ngraph.hpp>
+#include "layer_transformation.hpp"
+
+namespace ngraph {
+namespace pass {
+namespace low_precision {
+
+class TRANSFORMATIONS_API ReduceBaseTransformation : public LayerTransformation {
```

Comment:
Could you add a documentation for transformations?

Reply:
Documentation added for ReduceBaseTransformation. Derived classes have the same behavior